### PR TITLE
Update branches_arent_just_for_birds.html

### PR DESCRIPTION
**Pull request by zg**
Amending a change as suggested by @muan

### CODE REVIEW

**Comment by jlord** [comment]
:sparkles:

**Comment by jlord** [comment]
So actually, I should have read over the guide better the first time, because these changes don't exactly fit the flow of the challenge.

The first part of the challenge asks users to use the command `git status` to see that they are on a branch with a name, `gh-pages`. This is because the repo they forked only has a `gh-pages` branch, so there is no need to `checkout` and create a new branch named `gh-pages` — in fact that probably will fail since you're _already_ on a branch named `gh-pages`. The purpose of this exercise is to introduce the `git-status` command and learn that you can verify your branch that way.

It is this paragraph that gives the instruction to create a new branch and this new branch should be in the correct format:

> Create a branch and name it "add-<username>", where 'username' is your username. For instance, "add-jlord". Branches are case-sensitive so name your branch exactly the way your GitHub name appears.

Basically they should not be creating a new branch named `gh-pages` as it now instructs. But we can make these first statements clearer so that
